ext/main.dart

```diff
@@ -334,7 +334,12 @@ class WebRequestRedirect {
     if (_errorMode) {
       urls.addAll(makeIpfsGlobs(settings.server));
     } else {
+      urls.addAll(makeIpfsGlobs('https://gateway.ipfs.io'));
       urls.addAll(makeIpfsGlobs('http://gateway.ipfs.io'));
+	  urls.addAll(makeIpfsGlobs('https://ipfs.io'));
+      urls.addAll(makeIpfsGlobs('http://ipfs.io'));
+      urls.addAll(makeIpfsGlobs('https://ipfs.pics'));
```

Comment:
I don't like having the ipfs.pics urls in there. The idea with the ipfs.io URLs is to use them as the standard way to link into ipfs that's also backwards compatible with the current web.